src/connector.rs

```diff
@@ -607,4 +629,32 @@ impl EthConnectorContract {
     fn check_used_event(&self, key: &str) -> bool {
         sdk::storage_has_key(&self.used_event_key(key))
     }
+
+    /// Get Eth connector paused flags
+    pub fn get_paused_flags(&self) {
+        let data = self.get_paused().try_to_vec().unwrap();
+        sdk::return_output(&data[..]);
```

Comment:
This is maybe cheating a bit by using the fact that `PauseMask` is a `u8`, but it seems unnecessary to go through borsh for data this simple.

```suggestion
        let mask = self.get_paused();
        sdk::return_output(&[mask]);
```

Reply:
I was thinking about it, but this will need to be changed if we change the type of `PauseMask` later. Also, we will need to have another check for the input length then (borsh verifies this, that there's no other input data left).
I'd prefer to use Borsh for consistency if you are ok with it.

Reply:
Yeah, makes sense to me. You're right that preferring consistency to efficiency is the better approach in the long run.

And if we are ever at the point we care about gas used by borsh vs directly returning the byte then we're probably in great shape anyway haha!